src/templates/next/components/InfoCols/InfoCols.tsx

```diff
@@ -19,7 +19,7 @@ const InfoBoxIcon = ({ icon }: { icon?: SupportedIconName }) => {
   const Icon = SUPPORTED_ICONS_MAP[icon]
   return (
     <div>
-      <Icon className="w-10 h-auto text-secondary" />
+      <Icon className="w-10 h-auto text-site-secondary" />
```

Comment:
yeeesh can help me change this to primary 😆  it's supposed to be the orange according to design haha, brown icons mad ugly

Reply:
Updated in 0eef8dc0c1dc9a95fa438b8f72335b322ced07f8.